crates/uv/src/commands/project/init.rs

```diff
@@ -87,7 +97,7 @@ pub(crate) async fn init(
     .await?;
 
     // Create the `README.md` if it does not already exist.
-    if !no_readme {
+    if !no_readme && project_kind != InitProjectKind::Script {
```

Comment:
Generally, I'd use `matches!` instead of `!=` here. I'm not sure why. cc @BurntSushi

Reply:
`matches!` is strictly more flexible since you don't need a `PartialEq` impl to have it work. But for this specific instance, I don't think there's any meaningful advantage to one over the other. I don't mind the `!=` here personally, especially if you aren't adding `PartialEq` impls just to make it work.